include/appbase/application.hpp

```diff
@@ -26,6 +26,11 @@ namespace appbase {
           * @return Version output with -v/--version
```

Comment:
This comment is now wrong.

Reply:
Fair enough, but I wanted to eventually refactor _that_ version() call too if this PR is accepted. Get rid of the set_version(), and make the version() not require a rerun of cmake as it does now. (this version_string() approach _does not_ require a rerun of cmake).